custom_components/adaptive_lighting/switch.py

```diff
@@ -872,7 +874,7 @@ async def turn_on(service_data):
             if len(service_datas) == 2:
                 transition = service_datas[0].get(ATTR_TRANSITION)
                 if transition is not None:
-                    await asyncio.sleep(transition)
+                    await asyncio.sleep(transition + self._send_split_delay / 1000.0)
```

Comment:
If there is no transition, the send_split_delay will be ignored.

Reply:
Good remark. I moved it out of the if